src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/entity/Referral.kt

```diff
@@ -50,8 +47,10 @@ data class Referral(
   var desiredOutcomesIDs: List<UUID>? = null,
   var completionDeadline: LocalDate? = null,
 
+  // required fields
   @NotNull @ManyToOne(fetch = FetchType.LAZY) val intervention: Intervention,
   @NotNull val serviceUserCRN: String,
-  @CreationTimestamp var createdAt: OffsetDateTime? = null,
-  @Id @GeneratedValue var id: UUID? = null,
+  @NotNull @ManyToOne(fetch = FetchType.LAZY) val createdBy: AuthUser,
+  @NotNull val createdAt: OffsetDateTime,
```

Comment:
This kinda reminded me that we'll at some point need to rename/copy these to `startedBy` and `startedAt`, to follow the domain language (not here, not now)